pkg/sql/show_zone_config.go

```diff
@@ -122,7 +122,7 @@ func getShowZoneConfigRow(
 		}
 	}
 
```

Comment:
If we remove the `if !p.ExecCfg().Codec.ForSystemTenant() {` check in `ShowZoneConfig`, we should "just" get introspection.